core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -56,13 +56,13 @@ object HealthMonitor {
     Props(
       new HealthMonitor(
         List(
-          (Agora, checkAgora(methodRepoDAO)),
-          (Cromwell, checkCromwell(executionServiceServers)),
-          (Database, checkDB(slickDataSource)),
-          (GoogleBilling, checkGoogleBilling(googleServicesDAO)),
-          (GoogleBuckets, checkGoogleBuckets(googleServicesDAO, bucketsToCheck)),
-          (GooglePubSub, checkGooglePubsub(googlePubSubDAO, topicsToCheck)),
-          (Sam, checkSam(samDAO))
+//          (Agora, checkAgora(methodRepoDAO)),
+//          (Cromwell, checkCromwell(executionServiceServers)),
+          (Database, checkDB(slickDataSource))
+//          (GoogleBilling, checkGoogleBilling(googleServicesDAO)),
+//          (GoogleBuckets, checkGoogleBuckets(googleServicesDAO, bucketsToCheck)),
+//          (GooglePubSub, checkGooglePubsub(googlePubSubDAO, topicsToCheck)),
+//          (Sam, checkSam(samDAO))
```

Comment:
for this special Rawls instance, which is dedicated to re-migrating entities, no need for any of these other health checks. This also helps to reduce logging volume.